src/Geno/C++/GUI/MainWindow.cpp

```diff
@@ -33,7 +33,7 @@
 #include <examples/imgui_impl_opengl3.h>
 #include <GL/glew.h>
 #include <GLFW/glfw3.h>
-#include <GLFW/glfw3native.h>
+//#include <GLFW/glfw3native.h>
```

Comment:
This is needed on Windows

Reply:
Ah whoops - meant to wrap that with an `#if`